worlds/noita/Items.py

```diff
@@ -27,12 +27,14 @@ def create_fixed_item_pool() -> List[str]:
     return list(Counter(required_items).elements())
 
 
-def create_orb_items(victory_condition: VictoryCondition) -> List[str]:
+def create_orb_items(victory_condition: VictoryCondition, extra_orbs: ExtraOrbs) -> List[str]:
     orb_count = 0
+    if victory_condition != VictoryCondition.option_greed_ending:
+        orb_count = extra_orbs.value
```

Comment:
Would it be bad if someone wanted extra orbs for the greed ending (for whatever reason)?

Reply:
Probably? It would be weird at least to want kolmi stronger. I guess it's not actually a big deal? I'll change it